src/devices/bus/rs232/xvd701.h

```diff
@@ -6,6 +6,19 @@
 #include "rs232.h"
 #include "diserial.h"
 
+enum jvc_xvd701_media_type : uint32_t
+{
+	JVC_MEDIA_VCD = 0,
+	JVC_MEDIA_DVD = 1,
+};
+
+enum jvc_xvd701_playback_status : uint32_t
+{
+	STATUS_STOP = 0,
+	STATUS_PLAYING = 1,
+	STATUS_PAUSE = 2,
+};
```

Comment:
Is it really a good idea to drop these constants in the global namespace?

Reply:
No, this was just something I missed when cleaning up the code I was using to debug interaction with the Twinkle driver. I've moved it to the class now.